### PR TITLE
Add ToolTip for OriginalSymbolName

### LINKED ISSUE #16003: Can't see previous name of symbol you are renaming if its too long
Was renaming a member, can't remember the exact wording of the previous symbol, so expected to be able to hover it to get a tooltip with the full name. However, there is no tooltip - can we add one? Or reduce the font size of this to reduce occurrence?

image

### PR DESCRIPTION
**Pull request by reaction1989**
Fixes #16003

### CODE REVIEW

**Comment by Pilchie** [comment]
Thanks for the PR - tagging @dotnet/roslyn-ide for Code review.

File: src/EditorFeatures/Core/Implementation/InlineRename/Dashboard/Dashboard.xaml [hunk 1]
```diff
@@ -101,7 +101,8 @@
                                FontSize="14"
                                MaxWidth="260"
                                VerticalAlignment="Center"
-                               HorizontalAlignment="Left">
+                               HorizontalAlignment="Left"
+							   ToolTip="{Binding Session.OriginalSymbolName}">
```
Comment (Pilchie) [inline src/EditorFeatures/Core/Implementation/InlineRename/Dashboard/Dashboard.xaml]: Spaces instead of tabs please.

**Comment by reaction1989** [comment]
@jasonmalinowski The test failed after i fixed the spaces. At the first place all test went through.

The Text on which the ToolTip is on is powered by this code
`public string HeaderText
        {
            get
            {
                return string.Format(EditorFeaturesResources.Rename_colon_0, Session.OriginalSymbolName);
            }
        }`
So it should not be a Problem to use `Session.OriginalSymbolName` directly

**Comment by dpoeschl** [comment]
@reaction1989 Does that mean you understand what went wrong with the tests / how to fix it?

**Comment by reaction1989** [comment]
No. But all tests were green with the first commit. The second changed nothing. Ps i can not look at those results

Gesendet von meinem Windows 10 Phone

Von: David Poeschl<mailto:notifications@github.com>
Gesendet: Donnerstag, 5. Januar 2017 20:17
An: dotnet/roslyn<mailto:roslyn@noreply.github.com>
Cc: Stefan Nikolei<mailto:stefan@nikolei.eu>; Mention<mailto:mention@noreply.github.com>
Betreff: Re: [dotnet/roslyn] Add ToolTip for OriginalSymbolName (#16212)


@reaction1989<https://github.com/reaction1989> Does that mean you understand what went wrong with the tests / how to fix it?

-

**Comment by reaction1989** [comment]
No. But all tests were green with the first commit. The second changed nothing. 

@dpoeschl i can not look at the test results of windows_eta_open_prtest

Jenkins shows me this all the time: 
image

**Comment by reaction1989** [comment]
@dpoeschl You were right with the leak!

image

**Comment by dpoeschl** [comment]
The credit for that one goes to @jasonmalinowski :-)

**Comment by dpoeschl** [comment]
The link to the failing integration test is gone, so if the integration test fails again this time then I'll take a look. Fingers crossed though.

**Comment by Pilchie** [comment]
@jasonmalinowski Care to re-review?  @reaction1989 can you re-target to `post-dev15`?

**Comment by reaction1989** [comment]
@Pilchie it is re-targeted to post-dev15

**Pull request merged**